src/Application.php

```diff
@@ -105,4 +100,32 @@ public function run()
 
         return $this;
```

Comment:
hm... why? Isn't `completeRequest` return `$this`? You have the same in `handleException` so I think it should be fine. Can you think about edge case when it will be an issue? Any test case?